### ISSUE TITLE
v360Updater fails to set projects to active when using SQL Server and upgrading from last stable(v3.5.1) 

### ISSUE THREAD

**Issue by r8tor** [DependencyTrack/dependency-track#463]
### Current Behavior:
Dependency will run the upgrade routine at first start when upgrading from version 3.5.1 using a MS SQL Server database. The database update will fail with an error: 

ERROR [UpgradeExecutor] Error in executing upgrade class: org.dependencytrack.upgrade.v360.v360Updater
com.microsoft.sqlserver.jdbc.SQLServerException: Invalid column name 'TRUE'.

### Steps to Reproduce:
1) Using a dependency Track version 3.5.1, configured with MS SQL Server database.
2) Stop the application. 
3) Replace the file dependency-track-embedded.war with the newly downloaded version 3.6.0. 
4) Start the application with whatever parameters you have configured. For me it looks like this: java -Xmx4G -Dalpine.application.properties=/opt/dependency-track/application.properties -jar dependency-track-embedded.war -context / -host 127.0.0.1 -port 8080

Watch the logs and the error will show up early during the application initialization, before it starts updating the NVD feeds. 

### Expected Behavior:
Application starts normally and upgrades the database to work with the new version 3.6.0./

### Environment:
  
  - Dependency-Track Version:3.60
  - Distribution: Executable WAR
  - BOM Format & Version: Not relevant here. But always used CycloneDX for all my projects.
  - Database Server: MSSQL
  - Browser: Firefox 69.0.1 on MACOS

### Additional Details:
11:23:43.247 INFO [Config] --------------------------------------------------------------------------------
11:23:43.254 INFO [Config] OS Name:      Linux
11:23:43.254 INFO [Config] OS Version:   3.10.0-957.27.2.el7.x86_64
11:23:43.254 INFO [Config] OS Arch:      amd64
11:23:43.255 INFO [Config] CPU Cores:    2
11:23:43.255 INFO [Config] Java Vendor:  Oracle Corporation
11:23:43.255 INFO [Config] Java Version: 8.0.222
11:23:43.255 INFO [Config] Java Home:    /usr/lib/jvm/java-1.8.0-openjdk-1.8.0.222.b10-0.el7_6.x86_64/jre
11:23:43.255 INFO [Config] Java Temp:    /tmp
11:23:43.255 INFO [Config] User:         dtrack
11:23:43.255 INFO [Config] User Home:    /home/dtrack
11:23:43.255 INFO [Config] --------------------------------------------------------------------------------
11:23:43.255 INFO [Config] Initializing Configuration
11:23:43.256 INFO [Config] Loading application properties from /opt/dependency-track/application.properties
11:23:43.258 INFO [Config] --------------------------------------------------------------------------------
11:23:43.258 INFO [Config] Application:  Dependency-Track
11:23:43.259 INFO [Config] Version:      3.6.0
11:23:43.259 INFO [Config] Built-on:     2019-09-29T04:24:12Z
11:23:43.259 INFO [Config] --------------------------------------------------------------------------------
11:23:43.259 INFO [Config] Framework:    Alpine
11:23:43.259 INFO [Config] Version :     1.6.1
11:23:43.259 INFO [Config] Built-on:     2019-09-27T22:38:45Z
11:23:43.259 INFO [Config] --------------------------------------------------------------------------------
11:23:43.278 INFO [RequirementsVerifier] Initializing requirements verifier
11:23:43.279 INFO [UpgradeInitializer] Initializing upgrade framework
11:23:43.279 INFO [Config] Expanding classpath to include: /opt/dependency-track/mssql-jdbc-7.4.1.jre8.jar
11:23:44.594 WARN [Datastore] Default type for java type of java.lang.Float was previously jdbc-type=FLOAT but this is not provided by the JDBC driver! Please report this to the DataNucleus developers
11:23:44.594 WARN [Datastore] Default type for java type of java.lang.Double was previously jdbc-type=FLOAT but this is not provided by the JDBC driver! Please report this to the DataNucleus developers
11:23:45.588 INFO [UpgradeExecutor] Upgrade class org.dependencytrack.upgrade.v360.v360Updater about to run.
11:23:45.588 INFO [v360Updater] Updating project active status. Setting all projects to active
11:23:45.598 ERROR [UpgradeExecutor] Error in executing upgrade class: org.dependencytrack.upgrade.v360.v360Updater
com.microsoft.sqlserver.jdbc.SQLServerException: Invalid column name 'TRUE'.
        at com.microsoft.sqlserver.jdbc.SQLServerException.makeFromDatabaseError(SQLServerException.java:262)
        at com.microsoft.sqlserver.jdbc.SQLServerStatement.getNextResult(SQLServerStatement.java:1624)
        at com.microsoft.sqlserver.jdbc.SQLServerStatement.doExecuteStatement(SQLServerStatement.java:868)
        at com.microsoft.sqlserver.jdbc.SQLServerStatement$StmtExecCmd.doExecute(SQLServerStatement.java:768)
        at com.microsoft.sqlserver.jdbc.TDSCommand.execute(IOBuffer.java:7194)
        at com.microsoft.sqlserver.jdbc.SQLServerConnection.executeCommand(SQLServerConnection.java:2979)
        at com.microsoft.sqlserver.jdbc.SQLServerStatement.executeCommand(SQLServerStatement.java:248)
        at com.microsoft.sqlserver.jdbc.SQLServerStatement.executeStatement(SQLServerStatement.java:223)
        at com.microsoft.sqlserver.jdbc.SQLServerStatement.executeUpdate(SQLServerStatement.java:711)
        at com.zaxxer.hikari.pool.ProxyStatement.executeUpdate(ProxyStatement.java:120)
        at com.zaxxer.hikari.pool.HikariProxyStatement.executeUpdate(HikariProxyStatement.java)
        at alpine.util.DbUtil.executeUpdate(DbUtil.java:146)
        at org.dependencytrack.upgrade.v360.v360Updater.executeUpgrade(v360Updater.java:44)
        at alpine.upgrade.UpgradeExecutor.executeUpgrades(UpgradeExecutor.java:87)
        at org.dependencytrack.upgrade.UpgradeInitializer.contextInitialized(UpgradeInitializer.java:72)
        at org.eclipse.jetty.server.handler.ContextHandler.callContextInitialized(ContextHandler.java:921)
        at org.eclipse.jetty.servlet.ServletContextHandler.callContextInitialized(ServletContextHandler.java:554)
        at org.eclipse.jetty.server.handler.ContextHandler.startContext(ContextHandler.java:888)
        at org.eclipse.jetty.servlet.ServletContextHandler.startContext(ServletContextHandler.java:357)
        at org.eclipse.jetty.webapp.WebAppContext.startWebapp(WebAppContext.java:1443)
        at org.eclipse.jetty.webapp.WebAppContext.startContext(WebAppContext.java:1407)
        at org.eclipse.jetty.server.handler.ContextHandler.doStart(ContextHandler.java:821)
        at org.eclipse.jetty.servlet.ServletContextHandler.doStart(ServletContextHandler.java:276)
        at org.eclipse.jetty.webapp.WebAppContext.doStart(WebAppContext.java:524)
        at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
        at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
        at org.eclipse.jetty.server.Server.start(Server.java:407)
        at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:110)
        at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:106)
        at org.eclipse.jetty.server.Server.doStart(Server.java:371)
        at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
        at alpine.embedded.EmbeddedJettyServer.main(EmbeddedJettyServer.java:93)
11:23:45.598 ERROR [UpgradeInitializer] An error occurred performing upgrade processing. com.microsoft.sqlserver.jdbc.SQLServerException: Invalid column name 'TRUE'.
11:23:45.616 INFO [PersistenceManagerFactory] Initializing persistence framework
11:23:45.617 INFO [Config] Expanding classpath to include: /opt/dependency-track/mssql-jdbc-7.4.1.jre8.jar
11:23:45.777 WARN [Datastore] Default type for java type of java.lang.Float was previously jdbc-type=FLOAT but this is not provided by the JDBC driver! Please report this to the DataNucleus developers
11:23:45.777 WARN [Datastore] Default type for java type of java.lang.Double was previously jdbc-type=FLOAT but this is not provided by the JDBC driver! Please report this to the DataNucleus developers

**Comment by r8tor** [comment]
Adding some more information after further analysis of the problem.

I noticed that all of my projects where disabled. So editing the project and re-enabling them is a quick workaround for the issue. 

I traced the root cause to the class dependency-track/src/main/java/org/dependencytrack/upgrade/v360/v360Updater.java
It tries to run the following statement which causes the error: 
private static final String STMT_1 = "UPDATE \"PROJECT\" SET \"ACTIVE\" = TRUE WHERE \"ACTIVE\" IS NULL";

It seems like this SQL Statement  is not valid on MS SQL Server.. 
The following query works for MS SQL Server instances: UPDATE PROJECT SET ACTIVE=1 WHERE ACTIVE IS NULL;
